packages/components/src/styles.css

```diff
@@ -31,3 +31,8 @@
         left: -4px;
     }
 }
+
+body {
+    padding: 0 !important;
+    margin: 0 !important;
+}
```

Comment:
Sadly this line can't stay, I don't have a clue what it does but touching the `body` with `!important` then export it through `@frontify/fondue-components/styles` isn't a good idea IMO